src/adapters/acala.ts

```diff
@@ -0,0 +1,234 @@
+import { Wallet } from "@acala-network/sdk/wallet";
+import { AnyApi, FixedPointNumber } from "@acala-network/sdk-core";
+import {
+  catchError,
+  combineLatest,
+  firstValueFrom,
+  map,
+  Observable,
+  of,
+} from "rxjs";
+
+import "@acala-network/types/argument/api-tx";
+import { ApiRx } from "@polkadot/api";
+import { SubmittableExtrinsic } from "@polkadot/api/types";
+import { ISubmittableResult } from "@polkadot/types/types";
+
+import { BaseCrossChainAdapter } from "../base-chain-adapter";
+import { ChainName, chains } from "../configs";
+import { ApiNotFound } from "../errors";
+import {
+  BalanceData,
+  BasicToken,
+  CrossChainRouterConfigs,
+  CrossChainTransferParams,
+} from "../types";
+
+const ACALA_DEST_WEIGHT = "5000000000";
+
+// export const acalaRoutersConfig: Omit<CrossChainRouterConfigs, "from">[] = [
+//   {
+//     to: "interlay",
+//     token: "INTR",
+//     xcm: {
+//       fee: { token: "INTR", amount: "21787589" },
+//       weightLimit: ACALA_DEST_WEIGHT,
+//     },
+//   },
+//   {
+//     to: "interlay",
+//     token: "IBTC",
+//     xcm: {
+//       fee: { token: "IBTC", amount: "72" },
+//       weightLimit: ACALA_DEST_WEIGHT,
+//     },
+//   },
+// ];
+
+export const karuraRoutersConfig: Omit<CrossChainRouterConfigs, "from">[] = [
+  {
+    to: "kintsugi",
+    token: "KINT",
+    xcm: {
+      // local tests in chopsticks indicate fees are around 250k (atomic units)
+      fee: { token: "KINT", amount: "300000000" },
```

Comment:
I'm not sure how karura determined the fee, but it might be based on actual prices. Since KINT prices are likely to fluctuate, let's set a bigger margin (e.g. a 10x factor)

Reply:
> I'm not sure how karura determined the fee, but it might be based on actual prices. Since KINT prices are likely to fluctuate, let's set a bigger margin (e.g. a 10x factor)

Done in commit [e0bebde](https://github.com/interlay/bridge/pull/51/commits/e0bebde19a90cb0ecbfa271cc521d96d31cb2377)